portfolio/src/main/webapp/index.html

```diff
@@ -2,17 +2,43 @@
 <html>
   <head>
     <meta charset="UTF-8">
-    <title>My Portfolio</title>
+    <title>Essence Cain</title>
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
   </head>
   <body>
     <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+      <h1>Hello!</h1>
+      <h2>I am Essence Cain</h2>
+      <h3>About Me</h3>
+      <p>I am a current Computer Science student at North Carolina A&T State University, where I have began to explore mobile development and UX/UI design.</p>
+      <p>I believe technology will have a strong impact in advancing our society and I want to play a key role by offering a different perspective from undereppresnted groups 
+          in STEM fields.</p>
+      <h3>Work Experience</h3>
+      <dl>
+          <img src ="/images/google_icon.jpeg"/>
```

Comment:
I would remove the space are the src

```
          <img src="/images/google_icon.jpeg"/>
```

Also, I would add an alt tag to your images. These improve the SEO ranking of your page and it helps people with disabilities (like ppl who use screen readers).

More context:
https://moz.com/learn/seo/alt-text

```
          <img src="/images/google_icon.jpeg" alt="Google Icon" />
```

Reply:
When I removed the space between img and src the images no longer appeared

Reply:
1.  The alt text should just always be there. No need to add or remove when hovering.

2.
Oops i meant after the src.

Currently it's
`<img src ="/images/google_icon.jpeg"/>` 

I deally it should look like
`<img src="/images/google_icon.jpeg" />`

Reply:
Done